modules/core/src/test/java/org/apache/ignite/internal/processors/cache/persistence/pagemem/BPlusTreePageMemoryImplTest.java

```diff
@@ -120,7 +120,8 @@
                 }
             },
             () -> true,
-            new DataRegionMetricsImpl(new DataRegionConfiguration(), cctx.metric(), NO_OP_METRICS),
+            new DataRegionMetricsImpl(new DataRegionConfiguration(), cctx.metric(), cctx.performanceStatistics(),
```

Comment:
Should we add `PerformanceStatistics` instance to ctx on line 68?

Reply:
done